build.sh

```
@@ -926,13 +927,25 @@ function ensure_certificates() {
-                > /dev/null 2>&1
-        )
+        if [[ "$cert_name_prefix" == ecdsa* ]]; then
+            OPENSSL_ERR=$(
+                openssl req -x509 -nodes -days 3650 \
+                    -newkey ec -pkeyopt ec_paramgen_curve:P-256 \
+                    -keyout "$local_key_file" \
+                    -out "$local_cert_file" \
+                    -subj "/O=WSO2/OU=${PRODUCT_NAME}/CN=localhost" \
+                    -addext "subjectAltName=DNS:localhost,IP:127.0.0.1" \
+                    > /dev/null 2>&1
+            )
```

Shellcheck (0.11.0)

[warning] This command substitution will be empty because the command's output gets redirected away.

(SC2327)